docs/user-guides/configuration.md

```diff
@@ -208,9 +208,9 @@ agent:
 
 #### Ingress
 
-Ingress for gateways can be configured by `gateway.{backup,filter,lb,meta}.ingress` field object.
```

Comment:
I think we shouldn't remove filter from Ingress section.

Reply:
oops, sorry, it's my mistake

Reply:
fixed :smile: 31f7d1a